.github/workflows/component_packages.yml

```diff
@@ -79,21 +88,21 @@ jobs:
             exit 1
           fi
 
-      - name: Set tag and skip_upload=true for testing
+      - name: Skip publish and use snapshot for testing
         if: ${{ ! inputs.pre-release }}
         run: |
-          git tag ${{ inputs.tag_name }}
-          echo SKIP_UPLOAD=--skip=publish >> $GITHUB_ENV 
-      
+          echo SKIP_UPLOAD="--skip=publish" >> $GITHUB_ENV
+          echo USE_SNAPSHOT="--snapshot" >> $GITHUB_ENV
```

Comment:
just triggered the e2e as this might be an issue. IIRC e2e are building test binaries with a specific tag and this will introduce someting like `x.x.x-snapshot`, discard it if the tests pass

Reply:
Reverted in the last commit: [b58d9b3](https://github.com/newrelic/newrelic-agent-control/pull/1860/commits/b58d9b3956cf929e9f04b59e1fcfe9bdc4ae07ac)